README.md

```diff
@@ -1 +1,54 @@
-# Capstone_first
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse 1st capstone
+
+> This is my personnal portfolio, feel free to check it out.
+
+
+## Built With
+
+- Html
+- Css
+- Js
+
+
+
+ ## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/camilevahviraki/Capstone_first.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone https://github.com/camilevahviraki/Capstone_first.git`.
+- go to the repository folder in your command prompt `Capstone_first.git`
+
+You can also [type this Link](https://camilevahviraki.github.io/Capstone_first/) to see 
+the live demo of my personnal Portfolio.
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@camilevahviraki](https://github.com/camilevahviraki)
+- Twitter: [@camilevahviraki](https://twitter.com/CamileVahviraki)
+- LinkedIn: [camilevahviraki](https://www.linkedin.com/in/camile-vahviraki-8180a6232/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design as required by the Creative Commons License in your README file.